app/views/support/schools/show.html.erb

```diff
@@ -17,21 +17,25 @@
   </div>
 </div>
 
-<%= render Support::SchoolDetailsSummaryListComponent.new(school: @school) %>
+<%= render Support::SchoolDetailsSummaryListComponent.new(school: @school, viewer: current_user) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= govuk_button_link_to t('page_titles.new_school_user'), new_support_school_user_path(school_urn: @school.urn) %>
+    <% if policy(User).new? %>
+      <%= govuk_button_link_to t('page_titles.new_school_user'), new_support_school_user_path(school_urn: @school.urn) %>
+    <% end %>
 
     <h2 class="govuk-heading-l">Users</h2>
 
     <% if @users.present? %>
       <% @users.each do |user| %>
         <div class="user">
           <h3 class="govuk-heading-m govuk-!-margin-bottom-1"><%= user.full_name %></h3>
-          <p class="govuk-body">
-            <%= govuk_link_to "Edit user<span class=\"govuk-visually-hidden\"> #{user.full_name}</span>".html_safe, edit_support_school_user_path(school_urn: @school.urn, id: user.id) %>
-          </p>
+          <% if policy(User).edit? %>
```

Comment:
```suggestion
          <% if policy(user).edit? %>
```

Is it worth passing in the user instance if we have it?

Reply:
done